python/lsst/pipe/tasks/calexpCutout.py

```diff
@@ -45,7 +45,7 @@ class CalexpCutoutTaskConfig(pipeBase.PipelineTaskConfig,
 
 class CalexpCutoutTask(pipeBase.PipelineTask):
     """Task for computing cutouts on a specific calexp given
-    positions and sizes of the stamps.
+    positions, widths, and heights of the stamps.
```

Comment:
I strongly suggest adding the words xspan and yspan here, to make it clearer which table columns are needed. For example:

given position, xspan (width), and yspan (height).